bladex/params.py

```diff
@@ -0,0 +1,287 @@
+            'skew': None,
+            'camber': None
+        }
+        self.nbasis = {
+            'chord': 10,
+            'pitch': 10,
+            'rake': 10,
+            'skew': 10,
+            'camber': 10
+        }
+        self.degree = {
+            'chord': 3,
+            'pitch': 3,
+            'rake': 3,
+            'skew': 3,
+            'camber': 3
+        }
+        self.npoints = {
+            'chord': 500,
+            'pitch': 500,
+            'rake': 500,
+            'skew': 500,
+            'camber': 500
+        }
+        self.deformations = {
+            'chord': np.zeros(self.nbasis['chord']),
+            'pitch': np.zeros(self.nbasis['chord']),
+            'rake': np.zeros(self.nbasis['chord']),
+            'skew': np.zeros(self.nbasis['chord']),
+            'camber': np.zeros(self.nbasis['chord'])
+        }
+
+    def _check_params(self):
+        """
+        Private method that is called while writing a parameter file.
+        - The method checks if the user specifies a radii array, otherwise it
+            raises exception.
+        - In case no values assigned to any of the remaining parameter arrays
+            then an array of zeros is assigned, in which its length is equal
+            to the radii array.
+        - Any array that is not numpy is converted to be one.
+        - In case the user specifies the parameter array values but not with
+            same length, then an exception is raised.
+        """
+        if self.radii is None:
+            raise ValueError('Array radii can not have a None value.')
+
+        if not isinstance(self.radii, np.ndarray):
+            self.radii = np.asarray(self.radii)
+
+        for param in ['chord', 'pitch', 'rake', 'skew', 'camber']:
+            # If any parameter is not inserted then assign that parameter
+            # with array of zeros. This is useful in case the user is
+            # interested in only one or few parameters, while not interested
+            # in (or does have information about) the remaining ones.
+            if self.parameters[param] is None:
+                self.parameters[param] = np.zeros(self.radii.shape[0])
+
+            # In case inserted parameters are not numpy arrays
+            if not isinstance(self.parameters[param], np.ndarray):
+                self.parameters[param] = np.asarray(self.parameters[param])
+
+            # check the case if user inserts inhomogeneous radial distributions
+            if not self.parameters[param].shape == self.radii.shape:
+                raise ValueError(
+                    'Array ' + param + ' must have same shape of array radii.')
+
+            # If inserted deformations array not correspond to nbasis
+            if not self.nbasis[param] == len(self.deformations[param]):
+                raise ValueError(
+                    param + ' deformations must correspond to nbasis.')
+
+    def read_parameters(self, filename='parameter_files/parameters.prm'):
```

Comment:
Don't put a folder in the default value of the filename since the folder could not exist.